pkg/apis/minio.min.io/v2/helper.go

```diff
@@ -921,3 +923,32 @@ func GetTenantKesImage() string {
 	})
 	return tenantKesImage
 }
+
+// GetMonitoringInterval returns how ofter we should query tenants for cluster/health
+func GetMonitoringInterval() int {
+	monitoringIntervalOnce.Do(func() {
+		monitoringIntervalStr := envGet(monitoringIntervalEnv, "")
+		if monitoringIntervalStr == "" {
+			monitoringInterval = DefaultMonitoringInterval
+		}
+		val, err := strconv.Atoi(monitoringIntervalStr)
+		if err != nil {
+			monitoringInterval = DefaultMonitoringInterval
+		} else {
+			monitoringInterval = val
+		}
+	})
+	return monitoringInterval
+}
+
+// GetTenantServiceURL gets tenant's service url with the proper scheme and port
+func (t *Tenant) GetTenantServiceURL() (svcURL string) {
```

Comment:
We can use the `MinIOServerHostAddress()` in `v2/helper.go`, i think we have a couple of functions that do the same at this point, will also take a look and try to unify all of them

Reply:
I updated this function to do that, with the schema response